modules/tree/main/index.coffee

```diff
@@ -82,22 +123,20 @@ class Tree
     @options = hx.merge.defined {
       hideDisabledButtons: false
       animate: true
-      renderer: (elem, data) -> hx.select(elem).html(data)
+      renderer: (elem, data) -> hx.select(elem).html(data.name || data)
       items: []
     }, options
 
-    @selection = hx.select(@selector).classed('hx-openable', true)
+    @selection = hx.select(@selector).classed('hx-tree hx-openable', true)
```

Comment:
Not sure why we weren't adding `hx-tree` here - figured it makes sense to add it